api/models/loomies.go

```diff
@@ -169,3 +169,23 @@ func GetNearWildLoomies(coordinates interfaces.Coordinates) ([]interfaces.WildLo
 
 	return loomies, nil
 }
+
+func ValidateLoomieExists(loomie_id string) (bool, error) {
```

Comment:
You can just return the error, if there is an error you can infer that the Loomie doesn't exists.